com.unity.netcode.gameobjects/Components/AssemblyInfo.cs

```diff
@@ -5,6 +5,6 @@
 [assembly: InternalsVisibleTo("Unity.Netcode.Editor.CodeGen")]
 [assembly: InternalsVisibleTo("Unity.Netcode.Editor")]
 [assembly: InternalsVisibleTo("TestProject.EditorTests")]
-[assembly: InternalsVisibleTo("TestProject.RuntimeTests")]
 #endif
+[assembly: InternalsVisibleTo("TestProject.RuntimeTests")]
```

Comment:
The reason for this change is that internal visibility needs to be available when running within the editor (i.e. desktop) and runtime (i.e. consoles).